docs/tutorials/custom-cluster-selector.md

```diff
@@ -0,0 +1,637 @@
+---
+layout: default
+title: "Customizing Kubernetes YAMLs to target specific cluster"
+permalink: /tutorials/customize-cluster-selector
+parent: Tutorials
+nav_order: 9
```

Comment:
Also change nav_order for below tutorials to 91 and 92.

Reply:
Done